docs/tokens.md

```diff
@@ -0,0 +1,63 @@
+# Tokens && Fees
+
+## Supported tokens
+
+RIF Rollup can be used to transfer RBTC and ERC20 tokens. The full list of currently supported tokens is available under
+this [link]() (opens new window).
+
+> DISCLAIMER: Listing of a token on RIF Rollup does not represent an endorsement for this token by the RIF Rollup team,
+> IOVLabs, or any other entity. We do not have competence and authority to analyze the token business model and smart
+> contract security guarantees of a particular token. Please use any token on your own risk and judgement.
+
+## Listing a token
+
+### as user
+
+To list a new token on zkSync, go the [tokens page] (opens new window)of the explorer and click the "Add New Token"
+button and follow the instructions.
+
+### as operator
+
+1.  Deploy your ERC20 contract and take note of its _address_
+    1.  You can use `zk deploy-erc20 new <name> <symbol> <decimals> [<implementation>]` which will output
+        ```json
+        {
+          "address": "0x186E485150dDAB799Be8e25f87D717dEc45a86e5",
+          "name": "ProRif",
+          "symbol": "PRIF",
+          "decimals": "18"
+        }
+        ```
+2.  In rollup run `zk run governance-add-erc20 new <address>`. this will:
+    1. collect fees (token listing fees) on the `TokenGovernance` contract if caller is not on the `tokenLister` list
+       and `listingFee > 0` (see [deployment](/docs/contract.md#deployment) and
+       [Governance](/docs/contract.md#governance) for details)
+    2. adds token to the `Governance` contract
+    3. emits `NewToken` event
+    4. The token symbol and decimals will be fetched from our internal token list based on data from CoinGecko. If your
+       token is not on Coingecko, it will be listed as ERC20-<Token ID> with 18 decimals. Please note, that if your
+       token was listed on Coingecko after 2021-07-14, it may not be present in our token list and so make sure to
+       contact the IOVLabs RIF Rollup team before listing the token. FIXME: 💣 @RIF-ROLLUP-CORE-TEAM we could probably
```

Comment:
Should we remove this `FIXME`?

Reply:
no I added this fixme as we have to first fix that behaviour (or create a task)

Reply:
removed